src/modules/project/pages/projectView/views/projectMainBody/sections/ShareProject.tsx

```diff
@@ -36,6 +42,27 @@ export const ShareProject = () => {
     localStorage.setItem(SHARE_PROJECT_CLOSED_STORAGE_KEY, 'true')
   }
 
+  const handleTwitterShareClick = () => {
+    const projectLink = `${window.location.origin}/project/${project?.name}`
+
+    const tweetKeys = [
+      'Check out my project in geyser: {{projectLink}}',
+      "We're on a mission to change the game and we need your help. Dive into our project on @geyserfund and let's make a difference. Are you in? {{projectLink}}",
+      "Dream with us! We're launching something incredible on @geyserfund and you're invited to be part of the journey. Join us! {{projectLink}}",
+      "Big news! We've just launched our latest project on @geyserfund and we need YOUR help to make it a reality. Ready to make a difference? {{projectLink}}",
+      "Join us in bringing our dream project to life on @geyserfund. Let's change things together! {{projectLink}}",
+      "Breaking news: We're shaking things up with our new project on @geyserfund! Check it out: {{projectLink}} #BreakingNews",
+      "We're transforming our dreams into reality with this project on @geyserfund. Your support matters: {{projectLink}}",
+      'Making a difference, one step at a time. Join our mission on @geyserfund and help us leave a lasting impact. Your support makes all the difference: {{projectLink}} #MakeADifference',
+    ]
+    const randomKey = tweetKeys[Math.floor(Math.random() * tweetKeys.length)] || ''
+    const tweet = t(randomKey)
+    const tweetWithProjectLink = tweet.replace('{{projectLink}}', projectLink)
+
+    const twitterUrl = `https://twitter.com/intent/tweet?text=${encodeURIComponent(tweetWithProjectLink)}`
+    window.open(twitterUrl, '_blank')
+  }
+
```

Comment:
Maybe create a reusable function for create tweet url. I can see us doing more of these in the future.

Reply:
Good point, would the `utils` folder be a good location for such function?

Reply:
eah, that's fair, for now, you could just create a utils.ts file closest to where it's used. If more such functions popup in the same level, we can turn it into a folder. Basically going for vertical split ( have files closest to where it's used).